Makefile-tests.am

```diff
@@ -287,6 +293,17 @@ tests/%-symlink-stamp: % Makefile
 	ln -sf "$${real_bin}" tests/$*; \
 	touch $@
 
+# See above comment on binding the tests to be either installed or not.
+if ENABLE_INSTALLED_TESTS
+installed_test_scripts = $(_installed_or_uninstalled_test_scripts)
+installed_test_programs = $(_installed_or_uninstalled_test_programs)
+check-local:
+	echo "NOTE: Installed tests are enabled; to run them, make install, then: gnome-desktop-testing-runner -p 0 libostree/"
```

Comment:
OK, I got confused by `check-local`. I thought I remembered that it overrode `check`, but it's actually supplemental - http://www.gnu.org/software/automake/manual/html_node/Extending.html#Extending.